test/policies/project_policy_test.rb

```diff
@@ -84,6 +84,59 @@ def setup
     assert_equal 22, scoped_projects.count
   end
 
+  test 'scope expired memberships' do
+    # assuming personal projects cannot be expired
```

Comment:
This comment can also be removed

Reply:
3a2aeb09e85eb371b5b577b45d56a3ecd01938fc